src/components/Search/SearchAutocomplete.tsx

```diff
@@ -73,8 +79,10 @@ const SearchAutocomplete: React.FC<{
       renderOption={option => {
         return <MenuItem region={option} zipCodeInput={zipCodeInput} />;
       }}
+      getOptionLabel={getOptionLabel}
```

Comment:
adding `getOptionLabel` makes it so that when you select a location, the name of that location populates the searchbar for a second before navigating to the next page. it might look smoother without this/how it is currently on prod

Reply:
making getOptionLabel return an empty string could nix the location name flicker, but not sure if that nixes the error you were seeing for metros